README.md

```diff
@@ -1,94 +1,187 @@
+        # https://juju.is/docs/sdk/constructs#heading--statuses
+        self.unit.status = ops.ActiveStatus()
+...
+```
 
-The framework provides a testing harness, so you can ensure that your charm
-does the right thing in different scenarios, without having to create
-a full deployment. Our [API documentation](https://ops.readthedocs.io/en/latest/#module-ops.testing)
-has the details, including this example:
+> See more: [`ops.PebbleReadyEvent`](https://ops.readthedocs.io/en/latest/index.html#ops.PebbleReadyEvent)
 
-```python
+Note that the `tests/test_charm.py` file imports `ops.testing` and uses it to set up a testing harness:
+
+```
+...
+import ops.testing
+...
 class TestCharm(unittest.TestCase):
-    def test_foo(self):
-        harness = Harness(MyCharm)
-        self.addCleanup(harness.cleanup)  # always clean up after ourselves
+    def setUp(self):
+        self.harness = ops.testing.Harness(MyNewCharmCharm)
+        self.addCleanup(self.harness.cleanup)
+        self.harness.begin()
+...
+```
 
-        # Instantiate the charm and trigger events that Juju would on startup
-        harness.begin_with_initial_hooks()
+> See more: [`ops.testing.Harness`](https://ops.readthedocs.io/en/latest/#ops.testing.Harness)
 
-        # Update charm config and trigger config-changed
-        harness.update_config({'log_level': 'warn'})
 
-        # Check that charm properly handled config-changed, for example,
-        # the charm added the correct Pebble layer
-        plan = harness.get_container_pebble_plan('prometheus')
-        self.assertIn('--log.level=warn', plan.services['prometheus'].command)
+Explore further, start editing the files, or skip ahead and pack charm:
+
+```
+charmcraft pack
 ```
 
+If you didn't take any wrong turn or simply left the charm exactly as it was, this should work and yield a file called `my-new-charm_ubuntu-22.04-amd64.charm` (the architecture bit may be different depending on your system's architecture). Use this name and the resource from the `metadata.yaml` to deploy your example charm to your local MicroK8s cloud:
+
+```
+juju deploy ./my-new-charm_ubuntu-22.04-amd64.charm --resource httpbin-image=kennethreitz/httpbin
+```
+
+Congratulations, you’ve just built your first Kubernetes charm using Ops!
+
+### Clean up
+
+Delete your Multipass VM:
+
+```
+multipass delete --purge my-vm
+```
+
+Uninstall Multipass: [Linux](https://multipass.run/docs/installing-on-linux) | [macOS](https://multipass.run/docs/installing-on-macos) | [Windows](https://multipass.run/docs/installing-on-windows). On Linux:
+
+```
+snap remove multipass
+```
 
-## Talk to us
+## Next steps
 
-If you need help, have ideas, or would just like to chat with us, reach out on
-the Charmhub [Mattermost].
+### Learn more
+- Read our [user docs](https://juju.is/docs/sdk/ops) (which include links to the [API reference](https://ops.readthedocs.io/en/latest/) and other guides showing Ops in action)
```

Comment:
Perhaps simpler to separate out the API reference into its own bullet point:
```suggestion
- Read our [user documentation](https://juju.is/docs/sdk/ops), which includes other guides showing `ops` in action
- Dig into the [`ops` API reference](https://ops.readthedocs.io/en/latest/)
```

Reply:
OK